README.md

```diff
@@ -128,6 +101,16 @@ server.listen(server.get('port'), () => {
 });
 ```
 
+## Default components
+
+The `Oy` namespace exposes the following components validated against email best practices: 
+
+```
+Table TBody TR TD Img A
+```
+
+If you want to circumvent this validation, you can use `Oy.Element` and pass the `tagName` prop to implement your own validated element.
```

Comment:
This doesn't quite make sense. If you want to circumvent the validation, you could just use HTML elements directly. You would use Oy.Element if you want custom validation.